custom_components/hubitat/hubitatmaker/hub.py

```diff
@@ -470,20 +482,13 @@ async def _api_request(self, path: str, method="GET") -> Any:
 
     async def _start_server(self) -> None:
         """Start an event listener server."""
-        # First, figure out what address to listen on. Open a connection to
-        # the Hubitat hub and see what address it used. This assumes this
-        # machine and the Hubitat hub are on the same network.
-        with _open_socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
-            s.connect((self.host, 80))
-            address = s.getsockname()[0]
-
         self._server = server.create_server(
-            self._process_event, address, self.port or 0, self.ssl_context
+            self._process_event, self.address, self.port or 0, self.ssl_context
```

Comment:
Make this `self.address or '0.0.0.0'` since `create_server` expects a string value.